qemu/tests/cfg/nvdimm.cfg

```diff
@@ -8,7 +8,7 @@
     only x86_64, ppc64le
     ppc64le:
         required_qemu = [5, )
-        only nvdimm_basic
+        only nvdimm_basic nvdimm_nvml
```

Comment:
Since nvdimm is only supported in rhel8 guest, could you help to limit it on ppc

Reply:
Sure, updated.